src/UI/App.jsx

```diff
@@ -349,20 +521,26 @@ export default class App extends React.Component {
 	}
 
 	showChat = () => {
+		Logger.debug("Show chat, re-registering device");
```

Comment:
```suggestion
		Logger.debug("Show chat, registering device");
```

En aangezien je niet altijd bij show chat m daadwerkelijk registreerd, klopt dit dan wel?

Reply:
Ja, hij gaat hem altijd proberen te registreren. In die subscribe functie kijkt ie of het daadwerkelijk nodig is dus in je logs krijg je dan 2 scenarios:

```
Show chat, registering device
Found same device information from localStorage, using that instead of registering a new device
```

of

```
Show chat, registering device
Device registered, { ... }
```